platform/src/main/java/org/stellar/anchor/platform/controller/Sep31Controller.java

```diff
@@ -11,13 +11,16 @@
 import org.stellar.anchor.api.exception.AnchorException;
 import org.stellar.anchor.api.sep.AssetInfo.Sep31TxnFieldSpecs;
 import org.stellar.anchor.api.sep.sep31.*;
+import org.stellar.anchor.platform.condition.ConditionalOnSepsEnabled;
 import org.stellar.anchor.sep10.JwtToken;
 import org.stellar.anchor.sep31.Sep31Service;
 import org.stellar.anchor.sep31.Sep31Service.Sep31CustomerInfoNeededException;
 import org.stellar.anchor.sep31.Sep31Service.Sep31MissingFieldException;
 
 @RestController
+@CrossOrigin(origins = "*")
 @RequestMapping("sep31")
+@ConditionalOnSepsEnabled(seps = {"sep31"})
```

Comment:
Do you think SEPs 12 and 10 should both be mandatory for SEP-31? I'm not sure either.